pom.xml

```diff
@@ -54,6 +54,7 @@
             <plugin>
                 <groupId>org.apache.sling</groupId>
                 <artifactId>sling-maven-plugin</artifactId>
+                <version>3.0.2</version>
```

Comment:
this is part of latest parent, rather update parent version.

Reply:
Thanks @kwin ! It looks like Jörg already integrated the updates for java 17 into master, so now I merged them in instead of this commit.